index.css

```diff
@@ -0,0 +1,147 @@
+body {
+  overflow: hidden;
+  background: #3b0e68;
+  background: radial-gradient(circle, #3b0e68 0%, #1a084e 100%);
+}
+
+.center-text {
+  text-align: center;
+  color: white;
+}
+
+.bottom-text {
+  position: fixed;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  padding: 100px;
+  color: white;
+  text-align: center;
+}
+
+.mic {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  color: #fff;
+  cursor: pointer;
+  /* Adding cursor style to indicate it's clickable */
+}
+
+.mic::before,
+.mic::after {
+  content: "";
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  border-radius: 100%;
+  z-index: 2;
+  box-shadow: 0 0 10px 10px #1c084f;
+}
+
+.mic::before {
+  width: 150px;
+  height: 150px;
+  background-color: #1a084e;
+}
+
+.mic::after {
+  width: 100px;
+  height: 100px;
+  background-color: #2f1e5f;
+  animation: circle-size 0.8s linear infinite alternate;
+}
+
+.mic-icon {
+  box-sizing: border-box;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%) scale(2);
+  display: block;
+  width: 8px;
+  height: 6px;
+  border-bottom-left-radius: 60px;
+  border-bottom-right-radius: 60px;
+  border: 1px solid;
+  border-top: 0;
+  margin-top: 10px;
+  z-index: 3;
+}
+
+.mic-icon::before,
+.mic-icon::after {
+  content: "";
+  display: block;
+  box-sizing: border-box;
+  position: absolute;
+}
+
+.mic-icon::before {
+  width: 1px;
+  height: 3px;
+  top: calc(100% + 1px);
+  left: 50%;
+  transform: translate(-50%, 0);
+  background-color: #fff;
+}
+
+.mic-icon::after {
+  border: 1px solid;
+  width: 4px;
+  height: 9px;
+  left: 50%;
+  top: -5px;
+  border-radius: 2px;
+  transform: translate(-50%, 0);
+}
+
+.mic-shadow {
+  width: 150px;
+  height: 150px;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  border-radius: 100%;
```

Comment:
Use CSS variables for color values to easily update colors across the stylesheet [medium]

- Generated by [LGTM](https://lgtm.pro)